packages/payments-plugin/src/stripe/stripe-client.ts

```diff
@@ -0,0 +1,17 @@
+import Stripe from 'stripe';
+
+/**
+ * Wrapper around the Stripe client that exposes ApiKey and WebhookSecret
+ */
+export class VendureStripeClient extends Stripe {
+
+    constructor(
+        public apiKey: string,
```

Comment:
It's a minor thing but from what I could tell `apiKey` doesn't need to be public as it's not referenced anywhere unlike `webhookSecret` [here](https://github.com/vendure-ecommerce/vendure/pull/2058/files#diff-6d83de7289055b3d0fddfd6ca1e7e3343c119d4f650173bfd30194c56c61f935R73).

Reply:
You're right, it can be private.